### ISSUE TITLE
[CHORE] Use LingoHub for managing localization

### ISSUE THREAD

**Issue by cardoso** [RocketChat/Rocket.Chat.iOS#885]
Since we just doubled our number of supported languages, it's gonna be pretty hard from now on to keep track of what we need and the contributions along with the rest of the project.

Let's use [LingoHub](http://lingohub.com)?

They offer an Open Source membership, but we need to contact them first.

**Comment by rafaelks** [comment]
@cardoso Yes! We're actually using LingoHub for our web/back-end development. We just need to sync with the same account, that would definitely make things easier. 👍

**Comment by artrmz** [comment]
We should also implement (add some bot?) that will check if all resources from storyboards are also translated. Right now there is few keys (that are permanently set in storyboard/xib) missing in localize file. I'll create PR soon with adding those keys but it's really painful to maintain if people forget to add those keys. We could translate (add localized strings to each storyboard) each storyboard separately but I think spreading resources like that will be even harder to maintain?

**Comment by rafaelks** [comment]
@artrmz Yes, adding a bot that checks that would be wonderful! Do you know some service that does that? About the storyboard strings: we actually shouldn't use it. We should concentrate all the strings into the `Localizable.strings` files for each language.
